silta/silta.yml

```diff
@@ -9,3 +9,19 @@ varnish:
 
 elasticsearch:
   enabled: true
+
+exposeDomains:
+  - hostname: feature-redirects-exposedomain.drupal-project-k8s.silta.wdr.io
+    ssl:
+      enabled: true
+      issuer: letsencrypt-staging
+
+nginx:
+  redirects: |
```

Comment:
If we do some preprocessing on these values, wouldn't it make more sense to list them as a yaml array/object so we can cleanly iterate through them?

Reply:
Only in case we actually do preprocessing on them. 

If not, i (personally) would be happy to keep it as a string because some really long boring lists of redirects would need a list item prefix `-` then.

I am not against defining a list too.